src/content/tutorials/en/intro-to-p5-strands.mdx

```diff
@@ -0,0 +1,744 @@
+---
+title: "p5.strands: Introduction to Shaders"
+description: An introduction to shaders using p5.strands
+category: p5-strands
+categoryIndex: 0
+featuredImage: ../images/featured/intro-to-p5-strands.jpg
+featuredImageAlt: An abstract, cosmic scene with particles
+relatedContent:
+  references:
+    - en/p5/p5shader
+    - en/p5shader/modify
+    - en/p5shader/inspecthooks
+    - en/p5/basecolorshader/
+    - en/p5/basematerialshader/
+    - en/p5/basestrokeshader/
+    - en/p5/basenormalshader/
+    - en/p5/basefiltershader/
+authors:
+  - Luke Plowden
+---
+
+import EditableSketch from "../../../components/EditableSketch/index.astro";
+import Callout from "../../../components/Callout/index.astro";
+
+## Introduction
+
+In this tutorial, we'll explore **p5.strands**, a new way of writing shaders using JavaScript in p5.js. Shaders are programs that run on the GPU to create visual effects that would be difficult to achieve with traditional rendering.
+
+While many shader effects *could* be created with the p5.js 2D renderer, shaders are more efficient for complex effects and can be consistently applied across multiple objects. Learning shaders is valuable for anyone interested in graphics programming.

```

Comment:
Minor:

I really like making clear who is this for and what skills it develops outside of p5!

"Learning shaders is valuable for anyone interested in graphics programming" - VFX? game development, lighting, textures? VR? Real-time processing of video data? I don't really know what makes sense to include in this list but 1 or 2 concrete examples to make this more real could be good?